Double check test and rule out bugs in apache & redis roles:
```
host = <testinfra.host.Host ansible://opensuse-15-owncloud>

    def test_owncloud_running_services(host):
        assert host.service("apache2").is_running
        assert host.service("mariadb").is_running
>       assert host.service("redis-server").is_running
E       AssertionError: assert False
E        +  where False = <service redis-server>.is_running
E        +    where <service redis-server> = <class 'testinfra.modules.base.SystemdService'>('redis-server')
E        +      where <class 'testinfra.modules.base.SystemdService'> = <testinfra.host.Host ansible://opensuse-15-owncloud>.service

tests/test_default.py:13: AssertionError
```


```
host = <testinfra.host.Host ansible://opensuse-15-owncloud>
    def test_owncloud_web(host):
        code = int(
            host.run("curl -s -w '%{http_code}' http://localhost/index.php/login -o /dev/null").stdout
        )
        body = host.run("curl -sX GET http://localhost/index.php/login").stdout
>       assert code == 200
E       assert 404 == 200
```

--- a/molecule/opensuse15/tests/test_default.py
+++ b/molecule/opensuse15/tests/test_default.py
@@ -10,7 +10,7 @@ testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
 def test_owncloud_running_services(host):
     assert host.service("apache2").is_running
     assert host.service("mariadb").is_running
-    assert host.service("redis-server").is_running
+    assert host.service("redis@default").is_running
 
 
 def test_owncloud_network(host):

